dev-docs/RFCs/v7.x/post-process-effect-rfc.md

```diff
@@ -0,0 +1,66 @@
+# RFC: Post-processing Effect Rendering
+
+* **Authors**: Jian Huang
+* **Date**: May. 2019
+* **Status**: For Review
+
+## Abstract
+Post-processing effect rendering is part of deck.gl effect rendering pipeline, focusing on screen space pixel manipulation by using ping-pong technique.
+
+## Design
+
+A post-processing effect is created from `PostProcessEffect` class and `Shader module`.
+```js
+import {convolution} from @luma.gl/Effects;
+import {PostProcessEffect} from @deck.gl/core;
+
+const postProcessEffect = new PostProcessEffect(convolution, {
+  kernel: convolution.KERNEL.EMBOSS,
+  kernelWeight: convolution.KERNEL.EMBOSS.reduce((sum, x) => sum + x, 0)
```

Comment:
Side question: wonder if the weight calculation can be performance automatically in the `convolution` module's `getUniforms` method?

Reply:
Good suggestion, will think about it and write a RFC on luma.gl side to talk about these shader module details.